packages/backend-update/src/config/env.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod'
+
+const envSchema = z.object({
+  PORT: z.coerce.number().default(3000),
+  NODE_ENV: z.string().default('development'),
+  DATABASE_URL: z
+    .string()
+    .default(
+      'postgres://postgres:password@localhost:5433/testnet-backend-update'
+    ),
+  COOKIE_NAME: z.string().default('testnet.cookie'),
+  COOKIE_PASSWORD: z
+    .string()
+    .default('testnet.cookie.password.super.secret.ilp'), // min. 32 chars
+  COOKIE_TTL: z.coerce.number().default(2630000), // 1 month
```

Comment:
At this moment, we set the cookie TTL to 1 month. Do we want this to be session based and have the user log in every time it opens the browser?

Reply:
I will ask the client on this.

Reply:
I got confirmation that 1 month TTL is ok for testnet. No need to change.